docs/modules/ROOT/pages/upgrades.adoc

```diff
@@ -96,18 +96,11 @@ trait IUpgradeable {
 
 == Proxies and Starknet
 
-A proxy contract is a contract that delegates function calls to another contract.
-This type of pattern decouples state and logic.
-Proxy contracts store the state and redirect function calls to an implementation contract that handles the logic,
-usually catching these calls with a fallback mechanism.
-This allows for different patterns such as upgrades, where implementation contracts can change but the proxy contract (and thus the state) does not;
-as well as deploying multiple proxy instances pointing to the same implementation.
-This can be useful to deploy many contracts with identical logic but unique initialization data.
+Proxies allow for different patterns such as upgrades as well as deploying multiple clones of the same contract. But Starknet achieves these two things in different ways.
```

Comment:
```suggestion
Proxies enable different patterns such as upgrades and clones, but since Starknet achieves the same in different ways is that there's no proxy support. **Proxies cannot be implemented in Starknet**.
```
what do you think?

Reply:
I don't like stating that Proxies can't be implemented, because they can, right? We can have it for specific functions as the example I shared for beacons, or even by defining a fallback function ourselves receiving a selector and a calldata. Proxies can be implemented, even if the standards don't match Ethereum ones 100 percent.

Reply:
i think that's a very stretch definition of proxies, but I see your point. i also understand the "fallback function" approach although that's hardly an actual fallback function (since it's not a catch-all one). If you don't like the statement, what about:

```suggestion
Proxies enable different patterns such as upgrades and clones. But since Starknet achieves the same in different ways is that there's no support to implement them.
```